crash_similarity.py

```diff
@@ -76,10 +77,28 @@ def get_stack_trace_for_uuid(uuid):
     return data['proto_signature']
 
 
-def train_model(corpus):
-    if os.path.exists('stack_traces_model.pickle') and \
-       os.path.exists('stack_traces_model.pickle.docvecs.doctag_syn0.npy'):
-        return gensim.models.Doc2Vec.load('stack_traces_model.pickle')
+def delete_old_models(current_date, force_train):
```

Comment:
It is better to have "retrain timeout" as a variable parameter in any case. That's why `retrain_timeout_expired` function exists.